### DISCUSSION TITLE
What is the meaning of @tag or @if ?

### DISCUSSION THREAD

**Discussion by opvexe** [cue-lang/cue#1845] [Q&A] [answered]
What is the meaning of @tag or @if ?
I don't see such a description in the documentation.

```
@if(prod)

package tags

foo: string @tag(foo)
```

<img width="781" alt="image" src="https://user-images.githubusercontent.com/26895243/183289534-73acf121-ef25-479b-bdaf-3bc7cad7bf8b.png">

**Comment by verdverm** [comment, marked as answer]
`cue help injection`

The first if is similar to go build tags, generally thay are called attributes
